app/src/main/java/io/neurolab/tools/Utils.java

```diff
@@ -0,0 +1,38 @@
+package io.neurolab.tools;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class Utils {
+
+    public static int countLines(String filename) throws IOException {
+        return countLines(new File(filename));
+    }
+    public static int countLines(File file) {
+        try {
+            InputStream is;
+            is = new BufferedInputStream(new FileInputStream(file));
+            byte[] c = new byte[1024];
+            int count = 0;
+            int readChars = 0;
+            boolean empty = true;
+            while ((readChars = is.read(c)) != -1) {
+                empty = false;
+                for (int i = 0; i < readChars; ++i) {
+                    if (c[i] == '\n') {
+                        ++count;
+                    }
+                }
+            }
+            is.close();
+            return (count == 0 && !empty) ? 1 : count;
+        } catch (Exception e)
+        {
+            e.printStackTrace();
+        }
+        return -1;
```

Comment:
Use finally block to close the stream and use try with resources. Use exception rather than returning -1

Reply:
@iamareebjamal try-with resources requires API level 19. But we need to support from API level 16. Other than that I am making the changes you have asked for.

Reply:
Not true - https://stackoverflow.com/a/24290875/3309666

Reply:
@iamareebjamal It seems that I have not updated the plugin hence my AS shows that error-warning.